analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -337,6 +337,18 @@ End Sub
 Private Sub WithFunc(Arg As Func(Of Integer))
 End Sub");
 
+    [TestMethod]
+    public void Analyze_TopLevelStatements() =>
+        VerifyTopLevelStatements(@"
```

Comment:
We prefer `"""` since we have it for these cases. But this will turn into a single line, so simple inline string literal will do the trick

Reply:
I'm confused a bit - this is not the state of the current final commit in the branch. 
This test is using `"""`

Reply:
""" is meant to be used in multiline code
https://github.com/SonarSource/sonar-dotnet/blob/6e39dbb4ca6fc1e51780bedf84549e03f9ceb086/analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.IsNull.cs#L286-L289

Reply:
Ok  - I think there was some issue with the diff - git.
The comment here looks like you left it in ```Initialize_TopLevelStatements``` which was already changed to used `"""`, but in the `Files_changed` tab, it looked like you left it at ```Initialize_LocalFunction_TopLevelStatements``` which is multiline.

Anyways - change was applied. This can be resolved.

Reply:
Line 301: I still see `@"` there